src/encoder/ffmpeg/codec.cpp

```diff
@@ -86,6 +90,7 @@ std::vector<Packet> Codec::encode(const shar::Frame& image) {
       // before doing anything else, but who trust docs?
       av_packet_unref(&packet);
       ret = avcodec_receive_packet(m_context.get(), &packet);
+      m_full_delay.Observe(std::chrono::duration<double, std::milli>(Clock::now() - image.timestamp()).count());
```

Comment:
1. Split expression into several lines
2. There is no reason to use `double` as underlying type for `duration`.
3. `std::chrono::duration<{integer}, std::milli>` already defined in `std`. It is called `std::chrono::milliseconds`